.env.example

```diff
@@ -0,0 +1,35 @@
+# Copy this file to .env and replace the placeholders below
+
+# items marked <REDACTED> should be obtained and substituted
+# items marked <CONFIGURE> you will need to set up - these should match asl-conductor .env values
+
+DATABASE_NAME: asl
```

Comment:
env file is like:

`# Copy this file to .env and replace the placeholders below

# items marked <REDACTED> should be obtained and substituted
# items marked <CONFIGURE> you will need to set up - these should match asl-conductor .env values

DATABASE_NAME=asl
DATABASE_USERNAME=postgres
DATABASE_PASSWORD=test-password
DATABASE_HOST=localhost

SQS_REGION=eu-west-2
SQS_ACCESS_KEY=test
SQS_SECRET=test
SQS_URL=http://localhost:4566/000000000000/asl-dev

S3_REGION=eu-west-2
S3_ACCESS_KEY=test
S3_SECRET=test
S3_BUCKET=asl-dev
S3_LOCALSTACK_URL=http://localhost:4566

TRANSPORT_KEY=2A7F9BDC8E6A54F1D0C5E3B92783AF65
TRANSPORT_IV=4F2A8D9B3E6C1F5A

KEYCLOAK_REALM=Asl-dev
KEYCLOAK_URL=https://sso-dev.notprod.homeoffice.gov.uk/auth
KEYCLOAK_CLIENT=asl-dev-connect
KEYCLOAK_SECRET=6cc478c0-8b86-410a-9cef-589b6a6b723a
KEYCLOAK_USERNAME=asl-resolver
KEYCLOAK_PASSWORD=m5FqYQ7oPrBS64DJo54m584N

EMAILER_SERVICE=http://localhost:8084
REGISTER_SERVICE=http://localhost:8080

JWT_SECRET=conductor
`